.github/workflows/main.yml

```diff
@@ -4,7 +4,7 @@ on: [ push ]
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Curious why we're specifying 18.04 here when other public workflows like the Ruby, Python, and Go libraries are using latest? I'm guessing this is to avoid the upgrade to 20.04 soon, but why only do that in the node library?

Reply:
I can't remember the reason I'm using 18.04 here. Let's keep all of them use same image ubuntu-latest.